documentation/modules/auxiliary/multidrop.md

```diff
@@ -0,0 +1,92 @@
+This module dependent on the given filename extension creates either a .lnk, .scf, .url, desktop.ini file which includes a reference to 
+the the specified remote host, causing SMB connections to be initiated from any user that views the file. This allows for NetNTLM hashes to be captured
+by a listening user.
+
+## Vulnerable Application
+
+Microsoft Windows
+
+## Verification Steps
+
+  1. Install the application
+  2. Start msfconsole
+  3. Do: ```use auxiliary/multidrop```
+  4. Customise Options as required
+  5. Do: ```run```
+  6. A file pointing back to the listening host will then be generated.
+  7. Configure auxiliary/server/capture/smb or similar to capture hashes.
+  8. Upload the document to an open share or similar and wait for hashes.
+
+## Options
+
+FILENAME - This option allows you to customise the generated filename and filetpye that is generated.

```

Comment:
Module options are usually **bold** and on a new line of their own. Also `filetype` is spelt incorrectly.

Example:

```md
**FILENAME**

This option allows you to customise the generated filename and filetype that is generated.

```

Also `This can be changed using set FILENAME` is kind of redundant.

Reply:
fixed